server/routes/components.test.ts

```diff
@@ -129,7 +129,7 @@ const testComponent = {
       swagger_docs: '/swagger-ui.html',
     },
   ],
-} as Component
+} as unknown as Component
```

Comment:
This is a capitulation to the complexity 😓  - it would be possible to extend Component type to correctly include product info but it is a load of code and only used in this one place